first_doc.py

```diff
@@ -0,0 +1,2 @@
+def adder(a,b):
+    return a+b
```

Comment:
It would be great if this was in a class so we could do both infix and prefix notation.